packages/amplication-client/src/Entity/EntityFieldList.scss

```diff
@@ -1,10 +1,14 @@
 @import "../style/index.scss";
 
 .entity-field-list {
+  &__container {
+    border-bottom: 1px solid #2c3249;
```

Comment:
please replace it with our variables instead of hard coded color

Reply:
Used the necessary variable✅